content/applications/marketing/surveys/time_random.rst

```diff
@@ -1,48 +1,335 @@
-==============================
-Timed and randomized questions
-==============================
+================
+Create questions
+================
 
-When creating a survey in Odoo, there are options to set a time limit on the survey and randomize
-the questions.
+When it comes to :doc:`creating and customizing surveys <../surveys/create>` in the Odoo *Surveys*
+application, the creation and customization of questions for that survey are of the utmost
+importance.
 
-Time limit
-==========
+Fortunately, Odoo provides numerous ways to configure tailored questions for any kind of survey.
 
-During a timed survey, participants must finish the survey within a specified period of time.
-A common use case for implementing a time limit is that it greatly reduces the chance of
-participants looking up responses via external resources (e.g. web search), and reduces the survey
-to a "closed book" testing environment.
+To access a list of *all* the questions that have been created in the database, navigate to
+:menuselection:`Surveys app --> Questions & Answers --> Questions`. There, users can view and modify
+any question from any survey.
 
-Find the :guilabel:`Survey Time Limit` setting in the :guilabel:`Options` tab of the survey form,
-under the :guilabel:`Questions` section.
+However, there is only one place in the Odoo *Surveys* application where survey questions can be
+created. To do that, navigate to a survey form, by going to :menuselection:`Surveys app --> New`, or
+by selecting any pre-existing survey from the :guilabel:`Surveys` page (:menuselection:`Surveys app
+--> Surveys`).
```

Comment:
As this is a lengthy doc (and I'm sure users will be super grateful that you're going into so much detail for each question type), but I'm wondering if there's a way to summarize the info in a chart with links. Something like my list of removal strategies and their features [here](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.html)

Since you provided a lot of details for people to jump-to sections in the header, and users can jump to sections using the side navigation bar. So, I think this suggestion is a small minor thing and can be adjusted in a follow-up PR for further improvement if you wish (:

For the purposes of THIS PR, feel free to ignore. Just wanted to make a comment (: